engine/config_integrations/alertmanager.py

```diff
@@ -1,38 +1,50 @@
 # Main
 enabled = True
-title = "Alertmanager"
+title = "AlertManager"
```

Comment:
keep "Alertmanager", this is how it called by the authors https://prometheus.io/docs/alerting/latest/alertmanager/